dev/connect-jvm-client-mima-check

```diff
@@ -34,20 +34,18 @@ fi
 
 rm -f .connect-mima-check-result
 
-echo "Build sql module, connect-client-jvm module and connect-client-jvm test jar..."
-build/sbt "sql/package;connect-client-jvm/assembly;connect-client-jvm/Test/package"
+echo "Build required modules..."
+build/sbt "assembly/package;connect-client-jvm/Test/package"
```

Comment:
In the future, we need to add mllib, protobuf and other modules, and use their classpath at the same time. This will lead to continuous expansion of the classpath, so instead, execute `assembly/package` and use the classpath of assembly